chat/message.py

```diff
@@ -62,7 +64,7 @@ def fetch(cls, message_id: str):
             raise SkygearChatException(response['error'])
 
         if len(response['result']) == 0:
-            raise SkygearChatException("no conversation found")
+            raise SkygearChatException("no messages found")
```

Comment:
use `SkygearChatException("no messages found", ResourceNotFound)` so that skygear-server will not return 500